server/service/hosts.go

```diff
@@ -1057,6 +1057,7 @@ func (svc *Service) RefetchHost(ctx context.Context, id uint) error {
 		return ctxerr.Wrap(ctx, err, "save host")
 	}
 
+	// TODO(android): add android to this list?
```

Comment:
I don't think we have a refetch concept for Android because it follows a pub-sub model instead.